packages/vscode-extension/lib/expo_router_helpers.js

```diff
@@ -6,6 +6,18 @@ export function computeRouteIdentifier(pathname, params) {
   return query ? `${pathname}?${query}` : pathname;
 }
 
+export function compareNavigationDescriptors(a, b) {
```

Comment:
```suggestion
export function areNavigationDescriptorsEqual(a, b) {
```
`compare` (to me) suggests a "comparator" (like the callback to [Array.sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#parameters)) that orders the arguments, not just checks for equality.